### PR TITLE
Dynamic roles and permissions: side by side with old possible commands

### PR DESCRIPTION
**Pull request by luontola**
#869 

This PR is to make sure the new dynamic role system works the same way as the old possible commands system. Following PRs will remove the old code.

### CODE REVIEW

File: src/clj/rems/workflow/permissions.clj [hunk 1]
```diff
@@ -0,0 +1,145 @@
+(ns rems.workflow.permissions
+  (:require [clojure.set :as set]
+            [clojure.test :refer [deftest is testing]]))
+
+(def ^:private conj-set (fnil conj (hash-set)))
```
Comment (cscwooller) [inline src/clj/rems/workflow/permissions.clj]: What is the purpose of this def?

Comment (luontola) [reply src/clj/rems/workflow/permissions.clj]: To produce a set when conj'ing nil.

Comment (cscwooller) [reply src/clj/rems/workflow/permissions.clj]: I would rather put it to utils or sth with more descriptive name.

Comment (luontola) [reply src/clj/rems/workflow/permissions.clj]: I think the name is already as descriptive as it is, and it's such a rarely needed operation that moving it elsewhere only complicates reading the code.

Comment (Macroz) [reply src/clj/rems/workflow/permissions.clj]: I think this is fine as it is now.

**Pull request merged**